app/presenters/tree_builder.rb

```diff
@@ -277,13 +278,12 @@ def x_build_node(object, pid, options)    # Called with object, tree node parent
     end
 
     # Process the node's children
-    if Array(@tree_state.x_tree(@name)[:open_nodes]).include?(node[:key]) ||
-       options[:open_all] ||
+    node[:expand] = true if Array(@tree_state.x_tree(@name)[:open_nodes]).include?(node[:key])
```

Comment:
`include?` returns a `bool` already so you can assign it. Or do you need `node[:expand]` to be `nil` in the `false` case?